src/Mod/Draft/DraftEdit.py

```diff
@@ -288,8 +309,9 @@ def mouseMoved(self, event_callback):
             pos = event.getPosition()
             node = self.getEditNode(pos)
             ep = self.getEditNodeIndex(node)
-            if ep != None: 
-                if self.overNode != None: self.overNode.setColor(COLORS["default"])
+            if ep != None:
+                if self.overNode != None:
```

Comment:
Testing for `None` uses the `is` operator.
```py
if ep is not None:
    if self.overNode is not None:
```

Reply:
i'll include it in next commit. Thanks!